nuget/release.sh

```diff
@@ -51,7 +51,7 @@ esac
 
 # Generate release notes for NuGet
 RELNOTES="$BUILDEXTRA"/nuget/ReleaseNotes.html
-RELNOTESMD="$BUILDEXTRA"/installer/ReleaseNotes.md
+RELNOTESMD="$BUILDEXTRA"/ReleaseNotes.md
```

Comment:
Maybe it would make sense to refactor the rendering code out into its own script in `installer/` first, and only then move the whole shebang in a subsequent commit?